bbb-common-web/src/main/java/org/bigbluebutton/api/RecordingService.java

```diff
@@ -49,10 +49,11 @@
 
     private static final Pattern PRESENTATION_ID_PATTERN = Pattern.compile("^[a-z0-9]{40}-[0-9]{13}\\.[0-9a-zA-Z]{3,4}$");
     
-    private static String processDir = "/var/bigbluebutton/recording/process";
-    private static String publishedDir = "/var/bigbluebutton/published";
-    private static String unpublishedDir = "/var/bigbluebutton/unpublished";
-    private static String deletedDir = "/var/bigbluebutton/deleted";
+    private String PROCESS_DIR = "/var/bigbluebutton/recording/process";

```

Comment:
These properties should be configurable in case of using mounted devices (and for running tests).